readme.cchtml

```diff
@@ -0,0 +1,84 @@
+<h1>Runestone Interactive - Author Toolkit Tutorial:</h1>
+<p>This author toolkit is built using <strong>Github Codespaces</strong>, which
+    are development spaces that are hosted on the cloud. From a Codespace, you
+    can edit your files in a Visual Studio Code environment from within your
+    browser, and any processing power required is taken from the host machine
+    in the cloud. We are using this platform to set up a way for Runestone
+    authors to quickly and easily update their textbooks from anywhere.</p>
+<p>First, create a Github account if you do not already have one.</p>
+<p>Then, navigate to the textbook's repository (EX:
+    https://github.com/jbrock1999h/aata), click the green "Use this template"
+    at the top right, and "Open in Codespace" in the dropdown menu. <em>Note:
+        This step will eventually be replaced by an "edit me" button being
+        placed in the textbook that would directly link to the textbook's
+        Codespace.</em></p>
+<p><em><img src="images/Tutorial/Green_Button.png" alt="" width="849"
+            height="992"></em></p>
+<p>Github will take a short period of time (usually not more than 5 minutes) to
+    create your Codespace. Once it boots up, you should see something like
+    this:<br><img src="images/Tutorial/Blank_Codespace.png" alt="" width="1218"
+        height="590"></p>
+<p>When this screen appears, the Codespace will still be working in the
+    background to run the textbook build (unless they have already been built
+    in your Codespace). This will take from a few seconds to a few minutes
+    depending on the size of your source material.</p>
+<div>
+    <div>We had initially hoped to create this Codespace such that all setup
+        would be done automatically, but we ran into some issues with changing
+        the automatic working directory for git. We need the origin to be
+        defined using the URL of the user's repository, but were unable to find
+        a way to automate the process of finding the URL from within
+        Codespaces. So, we need the user to manually publish their fork and set
+        their origin remote so that their work later down the line is committed
+        to the correct branch of the repository. First, you will need to go
+        into the "Source Control" tab on the left side of your screen and click
+        the "publish" button:</div>
+    <div><img src="images/Tutorial/Publish.png" alt="" width="377"
+            height="392"></div>
+    <div>A prompt will come up saying that you do not have permission to push
```

Comment:
See https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=cli#forking-a-repository. You can fork the upstream repo from the CLI then run the appropriate git commands from the `postCreateCommand.sh` script.